core/src/main/java/bio/terra/pearl/core/dao/file/ParticipantFileDao.java

```diff
@@ -0,0 +1,63 @@
+package bio.terra.pearl.core.dao.file;
+
+import bio.terra.pearl.core.dao.BaseJdbiDao;
+import bio.terra.pearl.core.model.file.ParticipantFile;
+import org.jdbi.v3.core.Jdbi;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Component
+public class ParticipantFileDao extends BaseJdbiDao<ParticipantFile> {
+    public ParticipantFileDao(Jdbi jdbi) {
+        super(jdbi);
+    }
+
+    @Override
+    protected Class<ParticipantFile> getClazz() {
+        return ParticipantFile.class;
+    }
+
+    public List<ParticipantFile> findBySurveyResponseId(UUID surveyResponseId) {
+        return jdbi.withHandle(handle ->
+                handle.createQuery("""
+                                select file.* from %s file
+                                inner join answer a on file.file_name = a.string_value and a.format = 'FILE_NAME'
+                                where a.survey_response_id = :surveyResponseId
+                                """.formatted(tableName))
+                        .bind("surveyResponseId", surveyResponseId)
+                        .mapTo(clazz)
+                        .stream()
+                        .toList()
+        );
+    }
+
+    public List<ParticipantFile> findByEnrolleeId(UUID enrolleeId) {
+        return findAllByProperty("enrollee_id", enrolleeId);
+    }
+
+    public void deleteByEnrolleeId(UUID enrolleeId) {
+        deleteByProperty("enrollee_id", enrolleeId);
+    }
+
+    public Optional<ParticipantFile> findByEnrolleeIdAndFileName(UUID enrolleeId, String fileName) {
+        return findByTwoProperties("enrollee_id", enrolleeId, "file_name", fileName);
+    }
+
+    public List<ParticipantFile> findAllByFileNameForEnrollee(UUID enrolleeId, List<String> participantFileNames) {
```

Comment:
Let's make the naming consistent.  Alternatively, keep it simple and just have the method that takes a list, since you can always just call it with a single entry
```suggestion
    public List<ParticipantFile> findByEnrolleeIdAndFileNames(UUID enrolleeId, List<String> participantFileNames) {
```